examples/EventListenerExample.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+require __DIR__ . '/../vendor/autoload.php';
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\HttpFactory;
+use OpenTelemetry\Contrib\Zipkin\Exporter as ZipkinExporter;
+use OpenTelemetry\EventHandler\Dispatcher;
+use OpenTelemetry\EventHandler\Events;
+use OpenTelemetry\SDK\Attributes;
+use OpenTelemetry\SDK\Trace\SpanProcessor\SimpleSpanProcessor;
+use OpenTelemetry\SDK\Trace\TracerProvider;
+
+//Register the event listener pair, The dispatcher here is a global Class
+Dispatcher::getinstance()->listen(Events::START, 'OpenTelemetry\EventHandler\Listener\StartSpanListener::handle');
```

Comment:
since those listeners shouldn't be in the SDK, perhaps a better example would be to just pass in a callable (an inline function) which shows it actually changing the span in some way.

Reply:
But, it would lead to coupling of Listener with Dispatcher right? Isn't the better way to implement is keeping listener logic separate?

Reply:
I don't think it does lead to any coupling. The dispatcher's listen event accepts a callable, and in this example, a callable is passed which demonstrates some dummy functionality. Nobody would use this example as-is, it's just illustrating how you can plug in functionality via an event.

Reply:
I agree with @brettmc .
A [Callable](https://www.php.net/manual/en/language.types.callable.php) would not lead to any coupling on the contrary. It would allow for a Listener to be a class or a simple lambda function, whatever suits the needs of the users best.